### PR TITLE
adding blog entry for mmrm development video

### PR DESCRIPTION
**Pull request by kkmann**
tested locally.

### CODE REVIEW

File: blog/mmrm-video-nov-2022/index.qmd [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Visualizing successful collaboration"
+author: "SWE WG"
+subtitle: "A visual time lapse of six months of work on the mmrm R package"
+description: Communicating the benefit of open collaboration can be hard - we used the "gource" tool to create a video based on the repositoy history of the mmrm R package.
```
Comment (BrianLang) [inline blog/mmrm-video-nov-2022/index.qmd:5]:
```suggestion
description: Communicating the benefit of open collaboration can be hard - we used the "gource" tool to create a video based on the repository history of the mmrm R package.
```

File: blog/mmrm-video-nov-2022/index.qmd [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+date: "2022-11-20"
+categories: [news]
+image: "mmrm-video.png"
+---
+
+It can be hard to communicate the benefit of open-source collaboration, 
+the effort that goes into it, and its enormous impact on data science to
```
Comment (BrianLang) [inline blog/mmrm-video-nov-2022/index.qmd:12]:
```suggestion
the effort that goes into it, and its enormous impact on data science, to 
```
to separate the list from the next clause "to a non-technical audience"

File: blog/mmrm-video-nov-2022/index.qmd [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+categories: [news]
+image: "mmrm-video.png"
+---
+
+It can be hard to communicate the benefit of open-source collaboration, 
+the effort that goes into it, and its enormous impact on data science to 
+a non-technical audience. 
+Visualizations can be very helpful here and we created a 
+[video](https://youtu.be/DAIR_rBV3zQ) showing the 
+first six months of development of the mmrm R package.
```
Comment (BrianLang) [inline blog/mmrm-video-nov-2022/index.qmd:16]:
```suggestion
first six months of development of the [`mmrm`](https://openpharma.github.io/mmrm/main/) R package.
```
a link to the website or GitHub?

File: blog/mmrm-video-nov-2022/index.qmd [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+community organization, code reviews, bug reporting, or answering questions, and
+promoting the package - 
+all of which are important to make such a project a long-term success.
+
+If you want to co-star in the next iteration of the video, check out the
+[mmrm git repository](https://github.com/openpharma/mmrm), get in touch and
+contribute! Our contribution guidelines can be found [here](https://openpharma.github.io/mmrm/main/CONTRIBUTING.html).
+
+We used [gource](https://gource.io/) to create the video following an 
+[example and guide](https://youtu.be/_ZUddQAhPFw) by Nan Xiao.
```
Comment (BrianLang) [inline blog/mmrm-video-nov-2022/index.qmd:37]:
```suggestion
[example and guide](https://youtu.be/_ZUddQAhPFw) by [Nan Xiao](https://nanx.me).
```

**Comment by BrianLang** [comment]
super cool!

**Comment by BrianLang** [comment]
Did you mean to assign this to me?

**Comment by danielinteractive** [comment]
@BrianLang yeah because you started the review already - then you can follow progress and approve later
(assigned for PRs = primary reviewer).

**Comment by kkmann** [comment]
ah, ok asigning reviewers makes sense x) thx for the sharp eye!

**Review by BrianLang — APPROVED** [review]
Thanks Kevin, 
this looks great now. 
Cheers,
Brian

**Comment by BrianLang** [comment]
> @BrianLang yeah because you started the review already - then you can follow progress and approve later
> (assigned for PRs = primary reviewer).

Ah I had thought that "assignment" was for people who are PR collaborators/developers, and "reviewers" are for reviewers. Once I was assigned to this PR I was no longer allowed to submit a review. I swapped it back and it allowed me to review/approve.

**Pull request merged**